disposable_email_blacklist.conf

```diff
@@ -12,6 +12,7 @@
 0wnd.org
 0x207.info
 1-8.biz
+1.emailfake.ml
```

Comment:
Please only reference `emailfake.ml` instead of each sub domains: `1.emailfake.ml`, `2.emailfake.ml`, ...